### PR TITLE
[BPF] prettier conntrack dump

### PR DESCRIPTION
**Pull request by tomastigera**
## Description

    calico-bpf conntrack dump has less verbose output
    
    Not to bother user with too much information, the output shows where a
    connection goes from and too. If there is a service invoplved in the
    middle, it shows that too. You can get the orifginal output using the
    --raw  option.
    
    UDP 172.18.0.1:35199 -> 239.255.255.250:1900  Age: 56.28257308s Active ago 53.279924595s
    TCP 172.18.0.9:57736 -> 172.18.0.7:30333 -> 10.65.0.2:8055  Age: 2.535576963s Active ago 493.47131ms SYN-SENT
    UDP 172.18.0.1:5353 -> 224.0.0.251:5353  Age: 57.284951727s Active ago 50.27747628s
    UDP 172.18.0.1:37213 -> 239.255.255.250:1900  Age: 59.283295671s Active ago 57.282576585s
    TCP 172.18.0.1:52424 -> 172.18.0.7:9099  Age: 58.300322297s Active ago 12.945516929s ESTABLISHED

    TCP 172.18.0.9:57162 -> 172.18.0.6:30333 -> 10.65.0.2:8055 external client, service forwarded to/from 172.18.0.6  Age: 6.107113931s Active ago 988.204797ms SYN-SENT

    [BPF] clean up stale ICMPv6 conntrack entries

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
ebpf: clean up stale icmp6 conntrack entries
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Review by sridhartigera — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
